python/servo/package_commands.py

```diff
@@ -153,14 +153,14 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
                 return e.returncode
         elif is_macosx():
 
-            dir_to_build = '/'.join(binary_path.split('/')[:-1])
+            dir_to_build = path.dirname(binary_path)
```

Comment:
Please split out all of the hacking string processing -> `os.path` function changes into a separate commit.